### PR TITLE
updates to PR merging guidelines 

### PR DESCRIPTION
**Pull request by TheFoxAtWork**
per issue #430 and co-chair/TL discussions

@lumjjb @ultrasaurus @JustinCappos @pragashj

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -23,40 +23,100 @@ Anyone interested in contributing should join the mailing list and other
...
 
-If you are new to the group, reviewing pull requests and commenting on issues
-is a great way to get involved!
+If you are new to the group, reviewing pull requests and commenting on issues is
+a great way to get involved!  We strongly encourage and support all our members
+to participate in anyway they can.
 
-Except for urgent or very small grammar or spelling fixes, we leave pull 
-requests open for at least 24 hours, so that others have the chance to 
+Except for urgent or very small grammar or spelling fixes, we leave pull
```
Comment (lumjjb) [inline CONTRIBUTING.md:30]: hmm perhaps a separate "merging pull requests section?"

And perhaps the definition of small changes can reference the "simple changes" section below?>

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -23,40 +23,100 @@ Anyone interested in contributing should join the mailing list and other
...
+requests open for at least 24 hours, so that others have the chance to
 review/comment.   
 
-Anyone who reviews a pull request should leave a note to let others know that 
-someone has looked at it.  Any change should have a +1 from someone else.
-Larger changes should have multiple reviewers.
+#### Favorable review
```
Comment (lumjjb) [inline CONTRIBUTING.md:34]: i'm not quite sure i understand the motivation of this section. are we adding a mechanism of favorable reviews for non write access members that acts as a non-official github approval?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -23,40 +23,100 @@ Anyone interested in contributing should join the mailing list and other
...
 * Additional Formatting
   * Headlines, page titles, subheads and similar content should follow sentence
     case, and should not include a trailing colon.
   * Paragraphs do not start with leading indent. 
-  * Wrap lines at 80 characters, except where it would break a link. No need to 
-    reformat the whole paragraph to make it perfect -- fewer diffs are easier for
-    reviewers.
+  * Wrap lines at 80 characters, except where it would break a link. No need to
+    reformat the whole paragraph to make it perfect -- fewer diffs are easier
+for reviewers.
```
Comment (lumjjb) [inline CONTRIBUTING.md:116]: nit : alignment ( we need to have a formatting script for this eventually....)

**Review by lumjjb — COMMENTED** [review]
Thanks for adding this!

File: CONTRIBUTING.md (hunk 2 above)
Comment (TheFoxAtWork) [reply CONTRIBUTING.md:34]: So we've always had this in some fashion.  Anyone can review a PR and +1 it to say it looks good.  It lets a person with merging power know someone besides them has at least looked it over.  
The intent of this is to provide context around what is considered a 'good' review of the PR.

**Review by pragashj — APPROVED** [review]
LGTM

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -23,40 +23,104 @@ Anyone interested in contributing should join the mailing list and other
...
+
+#### Simple changes
+
+Simple changes are defined as:
+* spelling, typo, grammar
+* clarifications, minor pdates
+
+A person without access, other than the PR author, can and _is_ encouraged to
+review a PR and comment/+1 that they have done a review and found it favorable.
+A person with access then may perform the merge.
```
Comment (ultrasaurus) [inline CONTRIBUTING.md:73]:
```suggestion
A person with access, including the PR author, may then perform the merge.
```

**Review by ultrasaurus — APPROVED** [review]
left a few comments and suggestions, nothing blocking
overall, these are great improvements!

File: CONTRIBUTING.md (hunk 2 above)
Comment (lumjjb) [reply CONTRIBUTING.md:34]: i think there may be a mechanisms to allow non-write users to do things like this with github bots, could be something to look at in the future.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -23,40 +23,104 @@ Anyone interested in contributing should join the mailing list and other
...
-Anyone who reviews a pull request should leave a note to let others know that 
-someone has looked at it.  Any change should have a +1 from someone else.
-Larger changes should have multiple reviewers.
+#### Favorable review
+
+A favorable review is determined by the contents of the PR complying with the
+contributing guide, the writing style, and agreement the contents align with the
+SIG's goals, objectives, and scope.  It is anticipated that PRs submitted, with
+the exception of spelling and grammar changes, have been discussed with members
+of the SIG.
```
[the remark this answers is not in the document]
Comment (TheFoxAtWork) [reply CONTRIBUTING.md:40]:
```suggestion
of the SIG via slack or issues.
```

**Pull request merged**